lib/mastodon/maintenance_cli.rb

```diff
@@ -175,8 +175,175 @@ def fix_duplicates
       @prompt.say 'Finished!'
     end
 
+    MEDIA_TYPES = %w(accounts custom_emojis media_attachments).freeze
+
+    option :domain, type: :string, required: true, desc: 'Target server domain'
+    option :only, type: :array, enum: MEDIA_TYPES, desc: 'Only process these media types'
+    option :force, type: :boolean, desc: 'Skip new remote_url check'
+    desc 'fix-remote-url FROM TO', 'Fix remote_url of existing media'
+    long_desc <<-DESC
+      Fix the remote_url of the existing media when the remote media server is changed.
+
+      It will automatically check if the media can be obtained from the changed URL. To avoid this, specify the --force option.
+    DESC
+    def fix_remote_url(from, to)
+      @prompt = TTY::Prompt.new
+
+      @from   = from
+      @to     = to
+      @domain = options[:domain]
+      @types  = options[:only] || MEDIA_TYPES
+      @force  = options[:force]
+
+      @prompt.say "Start fixing the remote_url of #{@types.join ', '} on the remote server \"#{@domain}\" from \"#{@from}\" to \"#{@to}\"."
+      raise 'User canceled' unless @prompt.yes?('Continue?')
+
+      fix_accounts_remote_url!
+      fix_custom_emojis_remote_url!
+      fix_media_attachments_remote_url!
+
+      Rails.cache.clear
+
+      @prompt.say 'Finished!'
+    end
+
     private
 
+    def fix_accounts_remote_url!
+      return unless @types.include? 'accounts'
+
+      @prompt.say 'Fixing accounts…'
+
+      unless @force
+        accounts_avatar_remote_url = ::Account.where(domain: @domain).where('avatar_remote_url like ?', "#{@from}%").reorder(avatar_updated_at: :desc).first&.avatar_remote_url&.sub(/^#{@from}/, @to)
+
+        return @prompt.warn  'There is no corresponding remote avatar image, so skip it.'      unless accounts_avatar_remote_url
```

Comment:
I'd suggest skipping instead of skip here and below

Reply:
What does that mean?
I can't interpret English like a native speaker, so please supplement with another expression.

Reply:
"There is no corresponding remote avatar image, so skipping it."

This version will give the feel that the script is still in process. Saying "skip it" sounds more like the script is telling the user to do something.

"Skipping because no corresponding remote avatar image could be found." is another way to put it.